jdaviz/configs/imviz/plugins/orientation/orientation.py

```diff
@@ -332,6 +341,12 @@ def add_orientation(self, rotation_angle=None, east_left=None, label=None,
             # default rotation:
             wrt_data = self.viewer.selected_obj.first_loaded_data
             if wrt_data is None:  # Nothing in viewer
+                msg = "Viewer must have data loaded to add an orientation."
+                if from_ui:

```

Comment:
Personally, I prefer more explicit name like `raise_exception=False`. From UI is just one of many possible reason why someone might want or not want an exception raised. But I'll leave it up to you.